index.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable no-unused-vars */
+import _ from 'lodash';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './style.css';
+import { setLocalStorage, updateLocalStorage } from './modules/local_storage.js';
```

Comment:
Since all the functions are in one single file and you are not using Webpack in this Repo it is better to delete all these unnecessary lines.

Reply:
:+1: